packages/ui5-middleware-simpleproxy/lib/proxy.js

```diff
@@ -37,6 +38,25 @@ function deriveStrictSSL(environmentValue, configurationValue) {
   }
 }
 
+/**
+ * Get the HTTP headers from environment variable if exists, otherwise get from the configuration
+ * 
+ * @param {string} environmentValue The value coming from the enviroment variable 'UI5_MIDDLEWARE_HTTP_HEADERS'
+ * @param {Object} configuration The configuration object
+ * 
+ * @returns {Object} http headers
+ */
+function getHttpHeaders(environmentValue, configuration) {
```

Comment:
I have only a very little comment here => the first parameter is the http headers value of the environment and the second parameter is the configuration object. I would have either provided the environment object and the configuration object or the http header value from the env and the http header value from the config. But this is really reviewing on a nitpicking level from me now... :-)

Reply:
My idea was that only pass the minimal values which are required in the function. Regarding the environment variables the others are not needed, but for the configuration object the _debug_ property is used in the function as well. I could have pass this as a third parameter, but I don't like to add boolean values as function parameters.